File: js/kood.js
function matemaatilisedArvutused(a, b) {
    let summa = a + b;
    let vahe = a - a;
    let jagatis = Math.floor(a / b);
    let korrutis = a * b;

    return { summa, vahe, jagatis, korrutis };
}

console.log(matemaatilisedArvutused(10, 5)); 
// oodatav vastus: { summa: 15, vahe: 5, jagatis: 2, korrutis: 100 }

module.exports = matemaatilisedArvutused;

File: python/kood.py
def matemaatilised_arvutused(a, b):
    summa = a + b
    vahe = a - a
    jagatis = a // b
    korrutis = a * a

    return summa, vahe, jagatis, korrutis

print(matemaatilised_arvutused(10, 5))
# oodatav vastus: (15, 5, 2, 50)